native-file-windows.txt

```diff
@@ -1,15 +1,3 @@
 [binaries]
 c = 'clang-cl'
 cpp = 'clang-cl'
-
-[properties]
-c_args = [
-	'-fcolor-diagnostics',
-	'-Wno-language-extension-token',
-	'-I ./src/lib/evil/pcre/',
-	'-Wfatal-errors',
-	'-W-doesnt-exists-no-language-extension-token-kakaka'
-	]
-cpp_args = [
-	'-fcolor-diagnostics',
-	]
```

Comment:
@JPTIZ could you explain why are you removing this?

Reply:
They were actually for tests and didn't work (I've opened an issue for that: https://github.com/mesonbuild/meson/issues/6783?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDc1MzMyMjUwMToxMTk4ODgxNg%3D%3D#issuecomment-599634681)